docs/site/HasMany-relation.md

```diff
@@ -92,11 +94,45 @@ as follows:
 // import statements
 class Customer extends Entity {
   // constructor, properties, etc.
-  @hasMany(() => Order, {keyTo: 'custId'})
+  @hasMany(() => Order, {keyTo: 'customerId'})
   orders?: Order[];
 }
 ```
 
+Add the source model's id as the foreign key property (`customerId`) in the
+target model.
+
+{% include code-caption.html content="/src/models/order.model.ts" %}
+
+```ts
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class Order extends Entity {
+  @property({
+    type: 'number',
+    id: true,
+    required: true,
+  })
+  id: number;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  name: string;
+
+  @property({
```

Comment:
@hacksparrow I think the fk property is defined with `@belongsTo` decorator, see https://loopback.io/doc/en/lb4/BelongsTo-relation.html#defining-a-belongsto-relation

Reply:
Thanks a lot @jannyHou. It makes sense to add the property via a corresponding relation than manually adding one. I have updated the doc accordingly.